Sources/SmithyReadWrite/WritingClosure.swift

```diff
@@ -5,5 +5,135 @@
 // SPDX-License-Identifier: Apache-2.0
 //
 
-public typealias WritingClosure<T, Writer> =
-    (T, Writer) throws -> Void
+import struct Foundation.Data
+import struct Foundation.Date
+import enum SmithyTimestamps.TimestampFormat
```

Comment:
This file contains writing closures that are based on the `SmithyWriter` (so they are shared by XML, JSON, and FormURL).